participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/service/UserRegistrationServiceImpl.java

```diff
@@ -119,7 +119,8 @@ public UserRegistrationResponse register(
         throw new ErrorCodeException(ErrorCode.PENDING_CONFIRMATION);
       }
 
-      EmailResponse emailResponse = generateAndSaveVerificationCode(existingUserDetails);
+      EmailResponse emailResponse =
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Fixed review comment.